src/resources/postcss/base/full/typography/_anchors.pcss

```diff
@@ -14,6 +14,10 @@
 			text-decoration: none;
 		}
 
+		&:focus {
```

Comment:
I wonder if we need to **_add_** these - are they inheriting from somewhere?

Reply:
Yeah they're being inherited from the grouping that's right above it. I did it this way to minimize the code footprint.

![image](https://github.com/the-events-calendar/tribe-common/assets/22029087/0985f768-8ddb-4d8b-893f-130d5cc8c614)